notebooks/ICESat-2_Cloud_Access/ATL10_V6_download_script.py

```diff
@@ -0,0 +1,558 @@
+#!/usr/bin/env python
```

Comment:
I'm assuming this is a copy and paste of a functional script to create a new file so all is well!